.github/workflows/icu_ci.yml

```diff
@@ -250,6 +250,29 @@ jobs:
           cd icu4c/source;
           make dist
 
+  # clang build with some options
+  clang-options-build-and-test:
+    strategy:
+      # "fail-fast: false" lets other jobs keep running even if the test breaks in some other options.
+      fail-fast: false
+      matrix:
+        build_option:
+         [ --enable-static, --enable-static --disable-shared ]
```

Comment:
My guess is that what's _probably_ going on here is that genrb runs with an LD_LIBRARY_PATH, which normally should probably include *libicudata.so*, but now we don't have that shared library any more, so we can't find the data.

Reply:
The bug is under the configuration of "--disable-shared" so regardless we have  libicudata.so, genrb will not include it.